### PR TITLE
Autogenerate superuser

### PR DESCRIPTION
**Pull request by JoselenCeciliaQ**
## Description:
We aim to minimize the manual steps required for VideoHeadline deployment by automating the creation of the base case with default configurations. To achieve this, we have added a call to create_super_user.py in the Dockerfile, which creates a superuser with a default organization. 

**create_super_user.py:** 
Purpose: This Python script is designed to create a superuser in a Django application by calling acreate_super_user.sh shell script.
Usage: The script expects three command-line arguments: <username>, <email>, and <password>. It then invokes the create_super_user.sh shell script with these arguments.

**create_super_user.sh:**
Purpose: This shell script is responsible for creating a Django superuser. It receives the username, email, and password as command-line arguments and uses them to create a superuser using Django's management command.
Usage: It is intended to be called by another script or command that provides the necessary arguments.

## Steps to test change: 
Run the command:
docker build -t video-headline .
docker run -it -e AWS_ACCESS_KEY_ID=<your_access_key> 
           -e AWS_SECRET_ACCESS_KEY=<your_secret_key> 
           -e AWS_SESSION_TOKEN=<your_session_token> 
           video-headline
complete the user data in the console: username, email and password

### CODE REVIEW

File: infrastructure/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+RUN curl "https://s3.amazonaws.com/session-manager-downloads/plugin/latest/ubuntu_64bit/session-manager-plugin.deb" -o "session-manager-plugin.deb" \
+    && dpkg -i session-manager-plugin.deb \
+    && rm -rf /var/lib/apt/lists/*
+
+ENV AWS_ACCESS_KEY_ID=""
+ENV AWS_SECRET_ACCESS_KEY=""
+ENV AWS_SESSION_TOKEN=""
+
+# CMD ["yarn", "cdk", "bootstrap", "--profile", "$AWS_PROFILE"] && \
```
Comment (santiagomintegui) [inline infrastructure/Dockerfile]: this comment is necessary?

File: infrastructure/lib/videoheadline-stack.ts [hunk 2]
```diff
@@ -288,17 +288,17 @@ export class VideoheadlineStack extends Stack {
         });
 
         // Creation of the Docker image
-        const vhImage = new DockerImageAsset(this, 'MyBuildImage', {
-            directory: join(__dirname, '../../'),
-            exclude: ['/infrastructure/cdk.out'],
-        });
+        // const vhImage = new DockerImageAsset(this, 'MyBuildImage', {
```
Comment (santiagomintegui) [inline infrastructure/lib/videoheadline-stack.ts]: this comment is necessary?

**Review by Franco0712 — APPROVED** [review]

File: pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+## Description:
```
Comment (NoeliaBentancor) [inline pull_request_template.md]: Why are we adding this file?

File: infrastructure/create_super_user.sh [hunk 4]
```diff
@@ -1,13 +1,15 @@
...
+password="$3"
+organization_id= 1 
+
 echo "
 from django.contrib.auth import get_user_model
 User = get_user_model()
 try:
-    User.objects.create_superuser('$username', '$email', '$password')
-    print('Superuser was created successfully')
+    User.objects.create_superuser(username='$username', email='$email', password='$password',organization_id='$organization_id')
```
Comment (NoeliaBentancor) [inline infrastructure/create_super_user.sh]: Delete quotes in id

**Review by lucilamoreno — APPROVED** [review]

File: infrastructure/Dockerfile [hunk 5]
```diff
@@ -12,10 +12,18 @@ RUN apt-get update && \
     apt-get install -y python3 python3-pip && \
     rm -rf /var/lib/apt/lists/*
 
+RUN apt-get update  \
```
Comment (NoeliaBentancor) [inline infrastructure/Dockerfile:15]: Why are we adding this?

File: infrastructure/create_super_user.sh [hunk 6]
```diff
@@ -1,13 +1,14 @@
...
+password="$3"
+
 echo "
 from django.contrib.auth import get_user_model
 User = get_user_model()
 try:
-    User.objects.create_superuser('$username', '$email', '$password')
-    print('Superuser was created successfully')
+    User.objects.create_superuser(username='$username', email='$email', password='$password',organization_id=1)
+    print('Superuser was created successfully: $username email: $email')
```
Comment (NoeliaBentancor) [inline infrastructure/create_super_user.sh]:
```suggestion
    print('Superuser was created successfully with name $username and email $email')
```

File: infrastructure/Dockerfile (hunk 5 above)
Comment (JoselenCeciliaQ) [reply infrastructure/Dockerfile:15]: Updating the package is typically done  to ensure that the subsequent installation commands (apt-get install, for example) are working with the most up-to-date information about available packages.

**Pull request merged**